### PR TITLE
fix: [CDS-68371]: rbac button used for freeze window bulk actions

### PR DESCRIPTION
**Pull request by satyam-harness**
#### Screenshots
before 
image

after
image

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by satyam-harness** [comment]
retrigger jest

**Comment by satyam-harness** [comment]
retrigger titlecheck

**Comment by udayvunnam** [comment]
How about disabling the global checkbox itself ?

**Review by udayvunnam — DISMISSED** [review]

File: src/modules/72-freeze-windows/components/BulkActions/BulkActions.tsx [hunk 1]
```diff
@@ -23,24 +26,42 @@ export const BulkActions: FC<BulkActionsProps> = ({ onDelete, onToggleFreeze })
 
   return selectedItems.length > 0 ? (
     <Layout.Horizontal spacing={'small'} flex={{ align: 'center-center' }}>
-      <Button
+      <RbacButton
         variation={ButtonVariation.SECONDARY}
         onClick={() => onToggleFreeze({ status: 'Enabled' })}
         text={getString('enable')}
         size={ButtonSize.SMALL}
+        permission={{
```
Comment (pankajsingh-harness) [inline src/modules/72-freeze-windows/components/BulkActions/BulkActions.tsx:34]: This can be common out in some variable and used in all places.

**Review by pankajsingh-harness — APPROVED** [review]

**Comment by satyam-harness** [comment]
retrigger jest

**Review by udayvunnam — APPROVED** [review]

**Comment by udayvunnam** [comment]
I guess the commit https://github.com/harness/harness-core-ui/pull/15770/commits/3a78df1367673bca6d8d97ad3cfbdddc1ff0e994 is still needed


since individual checkboxes can be selected and then do bulk action?

**Pull request merged**